server/app/uk/gov/ons/addressIndex/server/controllers/AddressController.scala

```diff
@@ -63,29 +62,28 @@ class AddressController @Inject()(esRepo: ElasticsearchRepository)(implicit ec:
 
       logger info s"#addressQuery parsed: postcode: ${tokens.postcode} , buildingNumber: ${tokens.buildingNumber}"
 
-      format.stringToScheme() match {
-        case PostcodeAddressFile(str) => searchPafAddresses(tokens)
-        case BritishStandard7666(str) => searchUnsupportedFormatReply
-        case UnsupportedScheme(str) => searchUnsupportedFormatReply
-      }
+      format.stringToScheme().map {
+        case PostcodeAddressFile(_) => searchPafAddresses(tokens)
+        case BritishStandard7666(_) => searchUnsupportedFormatReply
+      }.getOrElse(searchUnsupportedFormatReply)
     }
 
   }
 
 
   private def searchPafAddresses(tokens: AddressTokens): Future[Result] = {
-    esRepo.queryAddress(tokens).map { addresses =>
-      Ok(Json.toJson(AddressBySearchResponseContainer(
-        AddressBySearchResponse(
-          tokens,
-          addresses = addresses.map(AddressResponseAddress.fromPafAddress),
-          limit = 10,
-          offset = 0,
-          total = addresses.size
-        ),
-        AddressResponseStatus.ok,
-        errors = Seq()
-      )))
+    esRepo.queryAddress(tokens).map {
```

Comment:
ah. I see now. Would be better as a case class, then just `map { resp =>  //blah }`